[DO NOT MERGE] Global Settings instructions changes (#1649)
cloudflare/cloudflare-docs: products/cloudflare-one/src/content/identity/devices/corp-device.md

</Aside>

1. On the [Teams dashboard](https://dash.teams.cloudflare.com), navigate to **My Team > Lists**.

> 1. Click **Create manual list** or **Upload CSV**. For larger teams, we recommend uploading a CSV or using Cloudflare's API endpoint.

Reviewer: ```suggestion
1. Click **Create manual list** or **Upload CSV**. For larger teams, it is recommended to upload a CSV or use Cloudflare's API endpoint.
```
